mamsdk.properties

```diff
@@ -0,0 +1,11 @@
+##### Enter MamSdk library maven url #####
+mamSdkLibraryMaven=https://raw.githubusercontent.com/citrix/citrix-mam-sdks/develop/maven
+
+##### Enter MamSdk library tools url #####
+mamSdkLibraryTools=https://github.com/citrix/citrix-mam-sdks/raw/develop/tools/java
+
+##### Enter MamSdk version number (e.g. 20.10.5.1) #####
+mamSdkVersion=20.10.5.1
+
+##### Enter App Package Name (package name has to be globally unique) #####
+appPackageName=com.citrix.mvpntestapp
```

Comment:
There's no newline at the end of this line.

Reply:
Yes, this should be fixed now